src/components/editions/pullquote.tsx

```diff
@@ -0,0 +1,128 @@
+import type { SerializedStyles } from '@emotion/core';
+import { css } from '@emotion/core';
+import { from } from '@guardian/src-foundations/mq';
+import { headline } from '@guardian/src-foundations/typography';
+import { SvgQuote } from '@guardian/src-icons/quote';
+import type { Format, Option } from '@guardian/types';
+import { map, withDefault } from '@guardian/types';
+import { pipe2 } from 'lib';
+import type { FC, ReactNode } from 'react';
+import { basePx, darkModeCss } from 'styles';
+import { getThemeStyles } from 'themeStyles';
+
+export const pullquoteWidth = '10.875rem';
+const pullquoteTailSize = '23px';
+
+const styles = (format: Format): SerializedStyles => {
+	const { kicker, inverted } = getThemeStyles(format.theme);
+	return css`
+		width: ${pullquoteWidth};
+		position: relative;
+		box-sizing: border-box;
+		padding: ${basePx(0, 1, 3, 1)};
+		margin: ${basePx(0.75)} ${basePx(2)}
+			calc(${basePx(2)} + ${pullquoteTailSize}) 0;
+
+		color: ${kicker};
+		border: 1px solid ${kicker};
+		border-top: 12px solid ${kicker};
```

Comment:
Prefer `rem`s over `px` (I think we probably make an exception for a `1px` border because it's still expected to be `1px` when scaled up or down, but `12px` _will_ be noticeable when scaling).